scripts/historical_sms_exporter.py

```diff
@@ -1,22 +1,31 @@
 import logging
 from datetime import datetime, timezone
 
+from brownie import chain
 from yearn.historical_helper import export_historical, time_tracking
+from yearn.networks import Network
 from yearn.treasury.treasury import StrategistMultisig
 from yearn.utils import closest_block_after_timestamp
 
 logger = logging.getLogger('yearn.historical_sms_exporter')
 
 def main():
     start = datetime.now(tz=timezone.utc)
-    # end: 2021-01-28 09:09:48 first inbound sms tx
-    end = datetime(2021, 1, 28, 9, 10, tzinfo=timezone.utc)
+    if Network(chain.id) == Network.Fantom:
+        # end: 2021-06-17 Fantom SMS deployed
+        end = datetime(2021, 6, 17, tzinfo=timezone.utc)
+        data_query = 'sms_assets{network="FTM"}'
+    else:
```

Comment:
be more strict here, currently the strict would assume eth for anything but fantom, which might lead to exporting incorrect data

Reply:
fix'd, commit pending